satpy/demo/fci.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Demo FCI data download."""
+
+import urllib
+import tarfile
+import pathlib
+from satpy import config
+
+_fci_uncompressed_nominal = (
+    "https://sftp.eumetsat.int/public/folder/UsCVknVOOkSyCdgpMimJNQ/"
```

Comment:
I hope so!  Maybe @ameraner or @sjoro can answer this?  If it's not guaranteed to stay online, then I'm not sure what we can do.

Reply:
Hi both, I will ask about this and let you know!

Reply:
So I asked around and the consensus is that the links will stay valid undefinitely. If new test data is released, it will get a new folder and the old one should be kept. When the data gets really old (several years), it might end up in the "MTG_old_test_data" folder. 
So I suppose that for the benchmarking it should be stable enough.

Reply:
Sounds great, thanks.